tools/c7n_azure/tests/templates/vmss.json

```diff
@@ -2,24 +2,28 @@
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
-        "virtualMachineScaleSets_cctest_name": {
-            "defaultValue": "cctest",
+        "virtualNetworks_cctestvmssVnet_name": {
+            "defaultValue": "cctestvmssVnet",
+            "type": "String"
+        },
+        "virtualMachineScaleSets_cctestvmss_name": {
+            "defaultValue": "cctestvmss",
             "type": "String"
         }
     },
     "variables": {},
     "resources": [
         {
-            "comments": "Generalized from resource: '/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/resourceGroups/tabarlow/providers/Microsoft.Compute/virtualMachineScaleSets/cctest'.",
+            "comments": "Generalized from resource: '/subscriptions/aa98974b-5d2a-4d98-a78a-382f3715d07e/resourceGroups/test_vmss/providers/Microsoft.Compute/virtualMachineScaleSets/cctestvmss'.",
```

Comment:
I don't think we need any of the comments in the ARM templates, especially the ones that say "Generalized from resource: subscriptions/<sub-id>" since this is open source and should probably not have our subscription ids hard coded anywhere. I know they were there already, but looking at it now just looks weird to me.

Reply:
I agree, but I don't think there is a reason to change it.
These URIs are also a part of recorded cassettes..